### PR TITLE
Document the swapping of StateScriptRetry configuration values

### PR DESCRIPTION
**Pull request by lluiscampos**
Changelog: None

Signed-off-by: Lluis Campos <lluis.campos@northern.tech>

### CODE REVIEW

**Comment by lluiscampos** [comment]
@drewmoseley please have a look.

**Comment by drewmoseley** [comment]
Looks ok to me.  @kacf how should we call this out? It seems like we really want to make sure users are aware of this change.

File: 05.Client-configuration/05.Configuration-file/50.Configuration-options/docs.md [hunk 1]
```diff
@@ -67,14 +67,14 @@ standard certificate trust chains.
...
+The timeout before a state script that previously returned the special retry
+return code (21), will be run again.
 
 #### StateScriptRetryTimeoutSeconds
 
-The timeout before a state script that previously returned the special retry
-return code (21), will be run again.
+The interval for which the script is allowed to keep retrying. After this
+interval has expired, another retry attempt will be treated as a failure and if
+possible, the update will be rolled back.
```
Comment (kacf) [inline 05.Client-configuration/05.Configuration-file/50.Configuration-options/docs.md]: I would add a "historical" note to both of these, which says that the behavior used to be swapped before Mender 2.0.0.

**Comment by kacf** [comment]
> how should we call this out? It seems like we really want to make sure users are aware of this change.

My plan was to use the changelog only, since it's a fairly detailed configuration setting. Do you think that's too low key?

File: 05.Client-configuration/05.Configuration-file/50.Configuration-options/docs.md (hunk 1 above)
Comment (lluiscampos) [reply 05.Client-configuration/05.Configuration-file/50.Configuration-options/docs.md]: :+1:

**Comment by lluiscampos** [comment]
New changes. Added the note for history reference.

**Comment by drewmoseley** [comment]
A note in the changelog is fine by me.

**Comment by lluiscampos** [comment]
FYI @drewmoseley, this will be the text in the Changelog:

MEN-2409: Swapped definition of StateScriptRetryTimeout and
StateScriptRetryInterval for the names to represent what they are
actually doing. This change breaks compatibility with current usage of
these configurable parameters. See documentation for correct usage.

**Pull request merged**